netbox/extras/urls.py

```diff
@@ -92,4 +92,6 @@
     path('scripts/results/<int:job_result_pk>/', views.ScriptResultView.as_view(), name='script_result'),
     re_path(r'^scripts/(?P<module>.([^.]+)).(?P<name>.(.+))/', views.ScriptView.as_view(), name='script'),
 
+    # Markdown
+    path('markdown/', views.MarkdownRenderView.as_view(), name="markdown_render")
```

Comment:
We should consider potential future use cases, where we might need to render other types of markup as well. I'd suggest creating a Markdown-specific view under a more general-purpose path:
```suggestion
    path('render/markdown/', views.MarkdownRenderView.as_view(), name="render_markdown")
```